plaso/cli/psteal_tool.py

```diff
@@ -186,9 +186,9 @@ def AnalyzeEvents(self):
     storage_reader = storage_factory.StorageFactory.CreateStorageReaderForFile(
         self._storage_file_path)
     if not storage_reader:
-      logger.error('Format of storage file: {0:s} not supported'.format(
-          self._storage_file_path))
-      return
+      raise errors.BadConfigOption(
+          'Format of storage file: {0:s} not supported'.format(
+              self._storage_file_path))
```

Comment:
add the "Format of storage file: {0:s} not supported for writing" check?

Reply:
I'm not sure what you mean here - this is only opened for reading.

Reply:
ack I thought it was the similar path as psort